style.css

```diff
@@ -0,0 +1,135 @@
+body {
+  margin: 0 auto;
+  font-family: Helvetica, sans-serif;
+}
+
+.zone {
+  color: #fff;
+  font-size: 2em;
+  border-radius: 4px;
+  transition: all 0.3s linear;
+  padding: 0;
+}
+
+/***********************************************************************
+       *  Nav Bar
+       **********************************************************************/
+
+.navbar {
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+  margin: 0;
+  font-size: 0.7em;
+}
+
+a {
+  color: #f5f5f6;
+  text-decoration: none;
+}
+
+a:hover {
+  color: #686de0;
+}
+
+li {
+  padding: 20px;
+}
+
+.sticky {
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  width: 100%;
+}
+
+#welcome-section {
+  background: gray;
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.welcome-section > h1 {
+  font-size: 3rem;
+  font-weight: 200;
+  color: #fff;
+}
+
+/***********************************************************************
+*  Project **********************************************************************/
+
+#projects {
+  text-align: center;
+}
+
+/***********************************************************************
+       *  Background color
+       **********************************************************************/
+
+.crimson {
+  background: crimson;
+}
+
+.flex-items {
+  display: flex;
+  flex-wrap: wrap;
+  background: #45567d;
+  padding-top: 3rem;
+  padding-bottom: 2rem;
+  margin-top: -4.7rem;
+}
+
+.images {
+  max-width: 400px;
+  min-width: 100px;
+}
+
+.images img {
+  width: 100%;
+}
+
+blockquote {
+  font-size: 20px;
+  color: white;
+}
+
+.show {
+  text-align: center;
+  font-size: 27px;
+  margin-top: -2rem;
+}
+
+.dark-bgr {
+  background: #303841;
+  text-align: center;
+  padding-top: 3rem;
+  margin-top: -1.4rem;
+}
+
+.dark-bgr h1 {
+  font-size: 40px;
+  color: white;
+}
+
+.dark-bgr p {
+  color: #bdbaba;
+  font-style: italic;
+}
+
+.links a {
+  padding: 1rem;
+  font-size: 20px;
+}
+
+footer {
+  display: flex;
```

Comment:
- [ ]  I think it would be better to wrap repetitive CSS code into a class to follow the DRY concept. 

`.flex {
display: flex;
}`